sdk/storage/azblob/ci.yml

```
@@ -0,0 +1,15 @@
+trigger:
+  paths:
+    include:
+      - sdk/storage/azblob
+
+pr:
+  paths:
+    include:
+      - sdk/storage/azblob
+
+
+stages:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
```

Azure Pipelines supports "absolute" pathing with template paths that correspond to the repo root. I prefer that approach since it prevents relative pathing issues (which you have here, since you are actually one directory deeper than the ci.yml file I assume this was copied over from).

```suggestion
  - template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
```

---

Ohh I see. I noticed similar changes in https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/ci.yml where CI is already running so I decided to give it a try. Let me change that and get back to you.